### ISSUE TITLE
loss increasing sharply when training goes further

### ISSUE THREAD

**Issue by xiaolinAndy** [snipsco/automatic-data-generation#23]
Hi, I have been trying to follow your work on SLU data augmentation. However, when I increase the training set size(4000) and the training epochs(50), the KL loss and reconstruction loss increase sharply. I wonder is it a bug of code or the reason of VAE model? Besides, I didn't change the parameters of optimizer or KL annealing weight. The loss curve looks as below
![TIM截图20191220162120](https://user-images.githubusercontent.com/21968240/71240721-d49c9000-2344-11ea-9f96-ba7af7719dbc.png)
